src/main/java/ch/njol/skript/conditions/CondIsDivisibleBy.java

```diff
@@ -0,0 +1,61 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Divisible By")
+@Description("Check if a number is divisible by another number.")
+@Examples("5 is divisible by 5")
```

Comment:
should also include an example that returns false for this
`11 is divisible by 10 # false`